app/managers.py

```diff
@@ -0,0 +1,42 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema_db.sqlite")
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            "INSERT INTO actors (first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> list:
```

Comment:
Indicate a list of what instances you return in the type annotation in the future